src/ui/templates/components/status-tag.njk

```diff
@@ -17,11 +17,11 @@
   {% endif %}
 
   {% if status === "Completed" %}
-    {% set classes = "govuk-tag--green" %}
+    {% set classes = "govuk-tag--turquoise" %}
```

Comment:
Is there a test which reflects this change?

Reply:
could do an e2e with for class test or Caroline to test it

Reply:
Thanks guys, good shout.

Typically we don't have E2E tests for "visual CSS things", since 99.9% of it is handled by the GOV design library and we don't have much/any customisation of the visual aspects.

However for status tags, I think it's important to have this.

I've got a new cypress command working for 1x scenario. `cy.checkClassName()`

Need to create or update dashboard and task lists E2E tests to consume this new command 👍

Reply:
Have updated RE the above with the latest commits.